### PR TITLE
update code according to MSRV change

### PR DESCRIPTION
**Pull request by Vrtgs**
## Motivation
there are a lot of things that are marked TODO that are waiting on MSRV that haven't been resolved

## Solution

update the code according to todo comments

### CODE REVIEW

**Comment by Vrtgs** [comment]
my bad I thought std::mem::size_of was added in the prelude of 1.70 😅, I was mistaken

File: tokio/src/runtime/scheduler/multi_thread/worker.rs [hunk 1]
```diff
@@ -1014,7 +1014,7 @@ impl Core {
             .tuned_global_queue_interval(&worker.handle.shared.config);
 
         // Smooth out jitter
-        if abs_diff(self.global_queue_interval, next) > 2 {
+        if self.global_queue_interval.abs_diff(next) > 2 {
```
Comment (Darksonn) [inline tokio/src/runtime/scheduler/multi_thread/worker.rs:1017]: Hmm, I think it reads better like this:
```suggestion
        if u32::abs_diff(self.global_queue_interval, next) > 2 {
```

Comment (Vrtgs) [reply tokio/src/runtime/scheduler/multi_thread/worker.rs:1017]: alright!

**Review by Darksonn — APPROVED** [review]
Thanks.

**Pull request merged**